### ISSUE TITLE
setting uncommented characters color

### ISSUE THREAD

**Issue by iskode** [spf13/spf13-vim#528]
Hi, 
new to vim settings process, I've installed spf13-vim but I find the text color of uncommented characters nearly similar to commented ones (grey). So I want to set the uncommented text color to white.
I've tried guifg=white in .vimrc.local but doesn't work.
Anyone could tell me how to do it?
Thanks in advance.

**Comment by BenBergman** [comment]
Did you just follow the [basic instructions](http://vim.spf13.com/#install) for installation?

Have you customized your vim configuration in any way other than installing spf13?

What type of file are you editing?

The default spf13 configuration should automatically turn on syntax highlighting and it should use the [solarized](http://ethanschoonover.com/solarized) colour scheme, which should have commented text in a very distinct colour compared to other text.

**Comment by iskode** [comment]
Thanks for your answer.
Yes I followed basic instructions. I'm on ubuntu 12.04, the olny customization I've done is to reduce a tab to 2 spaces by adding the following in the ~/.vimrc.local file:
   set shiftwidth=2 
   set tabstop=2 
   set softtabstop=2
I'm editing a ruby file ".rb, at the bottom is a sample of that file.
That's what it looks like even if I use the solarized color by putting this in the ~/.vimrc.local file:
  colorscheme solarized
What I want is just to let the commented text in its color and change the uncommented one in white color.
Any hint on how to accomplish this.
Thank you again for helping me.

![capture du 2014-01-14 13 41 43](https://f.cloud.github.com/assets/4013555/1910634/61df9e74-7d1c-11e3-9e6f-979dbbe5d650.png)
"

**Comment by BenBergman** [comment]
It looks like everything is all setup correctly on your system. The colours
however look like they are the `gnome-terminal`/Ubuntu default colours.

You could install the Solarized colour scheme with something like [this](https://github.com/sigurdga/gnome-terminal-colors-solarized), just use gvim
instead of vim (assuming colours are fine there), or you could change
colour schemes. Solarized will usually look weird, as it does for you, if
the colour pallette is not correct.

If you do like how Solarized currently looks for you except for comments,
you can change the foreground colour for the comments highlight group, but
I feel like the other solutions are more correct.

Let me know if any of those fix your problem.

**Comment by iskode** [comment]
Hi thank you for helping me.
I've tried gvim and I found it awesome because settings can be done interactively
it's from far better than scrolling an entire file for understand how to modify a single property.
So I changed directly the color template
to like this one at bottom which is just clear for me, I can distinuish clearly between commented and uncommented text. So I didn't resume install processes you suggested
Thanks a lot for your support!
![capture du 2014-01-16 10 59 13](https://f.cloud.github.com/assets/4013555/1929280/20df485a-7e95-11e3-97a6-98445d92fa31.png)
